data-prepper-plugins/prometheus-sink/build.gradle

```diff
@@ -0,0 +1,27 @@
+
+dependencies {
+    implementation project(':data-prepper-api')
+    implementation libs.armeria.core
+    implementation project(path: ':data-prepper-plugins:common')
+    implementation 'com.arpnetworking.metrics:prometheus-remote-protocol:1.0.1'
+    implementation project(':data-prepper-plugins:aws-plugin-api')
+    implementation 'io.micrometer:micrometer-core'
+    implementation 'com.fasterxml.jackson.core:jackson-core'
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'org.apache.commons:commons-compress:1.21'
+    implementation 'joda-time:joda-time:2.11.1'
+    implementation project(':data-prepper-plugins:common')
+    implementation project(path: ':data-prepper-plugins:common')
+    implementation 'com.fasterxml.jackson.core:jackson-core'
+    implementation 'org.hibernate.validator:hibernate-validator:7.0.5.Final'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
+    implementation 'software.amazon.awssdk:auth'
+    implementation 'org.apache.commons:commons-lang3:3.12.0'
+    implementation project(':data-prepper-plugins:failures-common')
+    implementation group: 'org.apache.httpcomponents.client5', name: 'httpclient5', version: '5.2'
```

Comment:
Please use the string notation here. You are using it on all the other dependencies. 

Also, are you using this or Armeria for the client? Do we need both?

Reply:
Thanks for the review @dlvenable , Fixed